packages/playwright-test/src/runner/processHost.ts

```diff
@@ -50,6 +51,7 @@ export class ProcessHost extends EventEmitter {
       detached: false,
       env: { ...process.env, ...this._extraEnv },
       stdio: inheritStdio ? ['ignore', 'inherit', 'inherit', 'ipc'] : ['ignore', 'ignore', process.env.PW_RUNNER_DEBUG ? 'inherit' : 'ignore', 'ipc'],
+      execArgv: experimentalLoaderOptions(),
```

Comment:
We do not need experimental options in non-TSM mode. I think we should check `process.env.PW_TS_ESM_ON` here to make it conditional.